lib/template/templates.c

```diff
@@ -325,7 +324,7 @@ log_template_options_defaults(LogTemplateOptions *options)
 {
   memset(options, 0, sizeof(LogTemplateOptions));
   options->frac_digits = -1;
-  options->ts_format = TS_FMT_ISO;
+  options->ts_format = -1;
   options->on_error = -1;
```

Comment:
This is set back to '-1' (change its value was a mistake...)
Please, approve the changes, until that we cannot merge this.